Header header

float64 x
float64 y

float64 size
float64 angle
float64 response
int32 octave
int32 class_id

